packages/odyssey-react/src/components/Toast/Toast.test.tsx

```diff
@@ -12,47 +12,64 @@
 
 import { act, render, fireEvent, screen, within } from "@testing-library/react";
 import Toast, { useToast } from ".";
-import type { Context } from ".";
 
 const role = "status";
 const title = "Toast";
 const body = "Descriptive body content (optional)";
 const id = "my-toast";
 
 describe("Toast", () => {
-  it("renders the toast", () => {
-    const { getByRole } = render(<Toast title={title} body={body} />);
+  it("renders the toast visibly", () => {
+    render(<Toast title={title} body={body} />);
 
-    expect(getByRole(role)).toBeInTheDocument();
+    expect(screen.getByRole(role)).toBeVisible();
   });
 
-  it("renders the toast title", () => {
-    const { queryByText } = render(<Toast title={title} body={body} />);
+  it("renders the toast title visibly", () => {
+    render(<Toast title={title} body={body} />);
 
-    expect(queryByText(title)).toBeInTheDocument();
+    expect(screen.getByText(title)).toBeVisible();
   });
 
-  it("renders the toast body", () => {
-    const { queryByText } = render(<Toast title={title} body={body} />);
+  it("renders the toast body visibly", () => {
+    render(<Toast title={title} body={body} />);
 
-    expect(queryByText(title)).toBeInTheDocument();
-  });
-
-  it("does NOT render the toast body if the title prop is not present", () => {
-    const { queryByText } = render(<Toast title={title} />);
-
-    expect(queryByText(body)).toBeNull();
```

Comment:
this test was never going to fail and I didn't see any corresponding logic in the component implementation so I'm removing it for now